src/screens/ModelPage/Outcomes/Outcomes.js

```diff
@@ -34,13 +34,13 @@ const Outcomes = ({
             } else if (rowLabel === '3 Months of Stay at home') {
```

Comment:
nit: I think this code is dead and/or wrong.

Per my comment about `durationDays` messing up the Outcome labels, "3 Months of" isn't supposed to be in `rowLabel`, so this code can't get hit.

Reply:
it's dead - i'll clean it up in a later refactor